_install-and-configure/configuring-opensearch/index-settings.md

```diff
@@ -172,6 +172,8 @@ OpenSearch supports the following dynamic index-level index settings:
 
 - `index.query.default_field` (List): A field or list of fields that OpenSearch uses in queries in case a field isn't specified in the parameters.
 
+- `index.query.max_nested_depth` (Integer): The maximum number of nesting levels for `nested` queries. Default is 20. Minimum is 1 (single `nested` query).
```

Comment:
```suggestion
- `index.query.max_nested_depth` (Integer): The maximum number of nesting levels for `nested` queries. Default is Integer.MAX_VALUE Minimum is 1 (single `nested` query).
```

Reply:
I own explanation to this one: introducing limits to previously unlimited setting is considered a breaking change. This is 100% fine for major version (like 3.0.0) but while backporting to `2.x` (2.13.0 in this case), we have to account for that and keep the default to "unlimited" (sort of), so the `20` should stay for 3.0 but `Integer.MAX_VALUE` in 2.13.0+

Reply:
@reta fair. But is this repo's `main` dedicated to `2.13` or `3.`?

Reply:
AFAIK we don't have any docs for OpenSearch `main` (aka 3.x), we usually tell the documentation team which version change goes to (but all pull requests are against `main`)  ...

Reply:
@reta Correct. All changes are against `main` and then we backport to the version it needs to be backported to. I'll add the 2.13 label